Prowlarr script typo

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

prowlarr

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/prowlarr.sh)"

### 📝 Provide a clear and concise description of the issue.

There is a typo in line 26 that causes the script to fail.
'curl -fsSL "https://github.tcom/Prowlarr/Prowlarr/releases/download/v${RELEASE}/Prowlarr.master.${RELEASE}.linux-core-x64.tar.gz" -o "$temp_file"'

The ".tcom' is unreachable, so the LXC fails to finish.

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 🔄 Steps to reproduce the issue.

Run the install command. 
bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/prowlarr.sh)"

### ❌ Paste the full error output (if available).

 ⠹curl: (6) Could not resolve host: github.tcom
 ⠸
[ERROR] in line 26: exit code 0: while executing command curl -fsSL "https://github.tcom/Prowlarr/Prowlarr/releases/download/v${RELEASE}/Prowlarr.master.${RELEASE}.linux-core-x64.tar.gz" -o "$temp_file"
curl: (22) The requested URL returned error: 400

### 🖼️ Additional context (optional).

_No response_

--- a/install/prowlarr-install.sh
+++ b/install/prowlarr-install.sh
@@ -23,7 +23,7 @@ mkdir -p /var/lib/prowlarr/
 chmod 775 /var/lib/prowlarr/
 cd /var/lib/prowlarr/ || exit
 RELEASE=$(curl -fsSL https://api.github.com/repos/Prowlarr/Prowlarr/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
-curl -fsSL "https://github.tcom/Prowlarr/Prowlarr/releases/download/v${RELEASE}/Prowlarr.master.${RELEASE}.linux-core-x64.tar.gz" -o "$temp_file"
+curl -fsSL "https://github.com/Prowlarr/Prowlarr/releases/download/v${RELEASE}/Prowlarr.master.${RELEASE}.linux-core-x64.tar.gz" -o "$temp_file"
 $STD tar -xvzf "$temp_file"
 mv Prowlarr /opt
 chmod 775 /opt/Prowlarr

